doc/index.rst

```diff
@@ -1,9 +1,32 @@
-The Plutus Platform
+Plutus Tools Developer Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
+Plutus Tools are a collection of off-chain infrastructure resources. Plutus Tools are built 
+for developers who are authoring web-based applications that implement smart contracts on the 
+Cardano blockchain. 
+
+The Plutus Tools Team supports the underlying resources that developers need who are 
+writing full applications using Plutus in Haskell, including off-chain code. 
+The term “off-chain code” refers to the part of a contract application’s code which 
+runs off the chain, usually as a contract application. 
+
+Because there are so many potential use cases for what you may choose to create, you are not 
+confined to only one prescribed way of using Plutus Tools. Instead, you can access 
+the entire tool box where you can find a collection of utilities and components. Pick and choose 
+from the tools and examples that best suit your use case. 
+
+The Plutus Platform
```

Comment:
I am not sure about the formulation of this section. It sounds like we have a platform, but I don't think that we really do.

For example I do not know what the Plutus Application Framework consists of, I am not sure we have any end-to-end use cases, etc.

Reply:
Thanks for your feedback. Do you think the info on this page is still largely accurate: 
https://plutus-apps.readthedocs.io/en/latest/plutus/explanations/platform.html

I think it may be important and useful to have a conversation about how we describe and characterize what Plutus Tools is. I have found this to be a challenging subject to have clarity about. 

I'd also like to learn more about the meaning of the content on these pages: 

https://plutus-apps.readthedocs.io/en/latest/reference/examples.html

https://github.com/input-output-hk/plutus-apps/blob/main/plutus-use-cases/ARCHITECTURE.adoc

Reply:
no, the stuff about the plutus platform is absolutely not accurate